zigbee2mqtt/set_config.py

```diff
@@ -89,8 +90,11 @@ def main(options_path, data_path):
 
     cfg.set_option('log_level', category='advanced')
     cfg.set_option('rtscts', category='advanced')
-    
-    cfg.set_option('soft_reset_timeout', category='advanced') 
+
+    cfg.set_option('soft_reset_timeout', category='advanced')
+
+    cfg.set_option('pan_id', catgory='advanced')
```

Comment:
a typo was merged!, catgory should be category

Reply:
fixed in https://github.com/danielwelch/hassio-zigbee2mqtt/commit/84df61b88bb52089effc889cf22ef4afd89507ca